cpu/cc2538/cc2538.lds

```diff
@@ -69,6 +69,9 @@ MEMORY
     SRAM (RWX) : ORIGIN = SRAM_START, LENGTH = SRAM_LEN
 }
 
+//End of HEAP, BEGIN is relative, see below.
```

Comment:
Comments in Contiki should be C-style, not C++. See [Contiki's code formatting rules](https://github.com/contiki-os/contiki/wiki/Code-Contributions#code-formatting).